features/order_confirmation.feature

```diff
@@ -0,0 +1,12 @@
+Feature: As a Visitor.
+  In order to know when my order is ready for pick-up.
+  After clicking buy now, I would like to see an order confirmation with estimated pick up time.
```

Comment:
The process to order is 
1. add dish to order
2. go to check out > gives a summary of order items
3. buy now > gives an order confirmation with est time for pickup

The order confirmation with pickup time comes after hitting "Buy now" and not after hitting "Checkout".